guides/guide/test.js

```diff
@@ -27,8 +27,9 @@ guide.step("Remove existing dependencies", function(){
 guide.step("Install donejs", function(){
   var branch = process.env.TRAVIS_PULL_REQUEST_BRANCH || process.env.TRAVIS_BRANCH
     || process.env.APPVEYOR_REPO_COMMIT || 'master';
+  var repo = process.env.TRAVIS_PULL_REQUEST_SLUG || process.env.TRAVIS_REPO_SLUG;
 
-	return guide.executeCommand("npm", ["install", "donejs/donejs#" + branch, "-g"]);
+	return guide.executeCommand("npm", ["install", repo + branch, "-g"]);
```

Comment:
I think here and in the other changed test it should be `repo + '#' + branch` because the SLUG variables do not provide the `#` separator.